packages/build/src/plugins_core/functions/index.js

```diff
@@ -112,27 +98,29 @@ const coreCommand = async function ({
 }) {
   const functionsSrc = relativeFunctionsSrc === undefined ? undefined : resolve(buildDir, relativeFunctionsSrc)
   const functionsDist = resolve(buildDir, relativeFunctionsDist)
-  const internalFunctionsSrc =
-    relativeInternalFunctionsSrc === undefined ? undefined : resolve(buildDir, relativeInternalFunctionsSrc)
+  const internalFunctionsSrc = resolve(buildDir, relativeInternalFunctionsSrc)
+  const internalFunctionsSrcExists = await pathExists(internalFunctionsSrc)
+  const functionsSrcExists = await validateFunctionsSrc({ functionsSrc, logs, relativeFunctionsSrc })
+  const [userFunctions = [], internalFunctions = []] = await getUserAndInternalFunctions({
+    functionsSrc,
+    functionsSrcExists,
+    internalFunctionsSrc,
+    internalFunctionsSrcExists,
+  })
 
-  if (functionsSrc !== undefined) {
-    if (!(await pathExists(functionsSrc))) {
-      logFunctionsNonExistingDir(logs, relativeFunctionsSrc)
-      return {}
-    }
+  if (functionsSrc && !functionsSrcExists) {
+    logFunctionsNonExistingDir(logs, relativeFunctionsSrc)
```

Comment:
We're still showing the warning if there is a `functionsSrc` configured but it doesn't exist. We're just not halting the bundling process anymore.